src/addons/dragAndDrop/backgroundWrapper.js

```diff
@@ -50,46 +50,6 @@ const propTypes = {
 }
 
 class DraggableBackgroundWrapper extends React.Component {
-  // constructor(...args) {
```

Comment:
are you sure you want to comment this component? I would uncomment it or delete if it isn't needed longer

Reply:
this was old comment , just i droped it